arm-network/2017-06-01/swagger/azureWAN.json

```diff
@@ -0,0 +1,2434 @@
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. ExpressRoute connection is not yet provisioned; follow the Location header to poll for final outcome."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            },
+            "patch": {
+                "operationId": "ExpressRouteConnections_Update",
+                "x-ms-examples": {
+                    "HubList": { "$ref": "../examples/ExpressRouteConnectionUpdate.json" }
+                },
+                "description": "Updates an ExpressRoute connection.",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/HubParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/GatewayParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ConnectionNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "parameters",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ExpressRouteConnection"
+                        },
+                        "description": "Parameters supplied to update an ExpressRoute connection."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Request successful. Returns the details of the updated gateway.",
+                        "schema": {
+                            "$ref": "#/definitions/ExpressRouteConnection"
+                        }
+                    },
+                    "202": {
+                        "description": "Request received successfully. ExpressRoute connection is not yet updated; follow the Location header to poll for final outcome."
+                    },
+                    "default": {
+                        "description": "*** Error Response: ***\n\n * 400 Invalid request."
+                    }
+                },
+                "x-ms-long-running-operation": true
+            },
+            "delete": {
+                "operationId": "ExpressRouteConnections_Delete",
+                "x-ms-examples": {
+                    "HubList": { "$ref": "../
```

Comment:
Are these resources (scalableVpnGateways and scalableExpressRouteGateways) created in the PUT of /hubs/{hubName}?